### PR TITLE
Add extra panel for 'The Science'

### LINKED ISSUE #16: Add another panel on homepage
We need one more panel on the homepage to have one panel each for:

* mechanobiology

* product

* applications

* science

### PR DESCRIPTION
**Pull request by mfurman20**
Added extra panel for 'The Science'

Resolves #16

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,8 +33,8 @@
     "develop": "gatsby develop",
     "format": "prettier --write '**/*.js'",
     "clean": "gatsby clean",
-      "test": "echo \"Error: no test specified\" && exit 1",
-      "deploy": "gatsby build --prefix-paths && gh-pages -d public"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "deploy": "gatsby build --prefix-paths && gh-pages -d public"
```
Comment (jidicula) [inline package.json:37]: In the future, small styling fixes like this should be separated into their own commit.

File: src/components/Three.js [hunk 2]
```diff
@@ -38,7 +38,7 @@ const Three = props => (
       </div>
     </Fade>
     <ScrollLink
-      to="four"
+      to="threetwo"
```
Comment (jidicula) [inline src/components/Three.js:41]: Naming should be fine like this for now, at some point we may want to rename each panel to correspond with its placement

File: package.json (hunk 1 above)
Comment (mfurman20) [reply package.json:37]: Not sure what this is...if I made that change it was just a sloppy accident!

File: src/components/Three.js (hunk 2 above)
Comment (mfurman20) [reply src/components/Three.js:41]: I think that's a good idea, can be a quick-fix issue

**Pull request merged**